src/types/response.rs

```diff
@@ -326,10 +397,10 @@ impl From<Edn> for EntityHistoryResponse {
     }
 }
 
-fn nullable_str(s: String) -> Option<String> {
-    if s.contains("nil") {
-        None
-    } else {
-        Some(s)
-    }
-}
+// fn nullable_str(s: String) -> Option<String> {
```

Comment:
Why is this commented now?

Reply:
some PR you made unsued this, I left it there to see how it goes without it

Reply:
Can we delete this code/comment?

Reply:
guess it is ok as nothing failed e3a9856